requirements-dev.txt

```diff
@@ -13,3 +13,6 @@ sphinx
 
 # For `make package`
 wheel
+
+# For datetime tests
+isodate
```

Comment:
The `setup.py` file lists dependencies that are necessary for Pulp Smash to function. In contrast, `requirements-dev.txt` lists dependencies that are necessary for extra tasks, such as generating documentation with Sphinx or running the linters. Given that this new requirements is necessary for Pulp Smash to function, please place it in setup.py.